### ISSUE TITLE
VCell GUI needs to be evaluated for Colorblindness

### ISSUE THREAD

**Issue by CodeByDrescher** [virtualcell/vcell#1605]
We need to verify that all views, menus, windows, etc. are colorblind-safe.

The obvious targets are model and results viewers but all areas should be evaluated.
